src/slider/slider.scss

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
+$track-height: $iui-xs;  // 4px
+$tick-height: $iui-s+$iui-xxs;
+
+@mixin iui-slider-component-container {
+  display: inline-flex;
+
+  span.iui-left,
+  span.iui-right {
+    @include iui-font-family;
+    user-select: all;
+    margin-top: $iui-xxs;
+
+    .iui-icon {
+      @include iui-icons-default;
+      margin-top: 4px;
+    }
+  }
+  
+  span.iui-left {
+    margin-right: $iui-s;
+    text-align: right;
+  }
+
+  span.iui-right {
+    margin-left: $iui-s;
+  }
+}
+
+@mixin iui-slider-container {
+  display: flex;
+  position: relative;
+  z-index: 1; // above ticks
+  width: 100%;
+  pointer-events: none;
+  flex-direction: column;
+  justify-content: center;
+
+  .iui-tooltip {
+    display: none;
```

Comment:
hmm.. I dont think this is needed in the css (can be in html). Cause when implementing, we will be handling the tooltip in react side anyway (so this should not be needed...)

Reply:
Don't you want your css generic so it can be use with or without iTwinUI-react?

Reply:
Maybe I want, but we might need to remove tooltip handling from this css, cause in React it will work and be applied differently and these rules might mess it up..